FlashEnglish/Sources/FlashEnglishApp.swift

```diff
@@ -10,6 +10,11 @@ import GoogleMobileAds
 import Firebase
 import FirebaseInstallations
 
+
+
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/26f8fd3d00e8b28f320403772b345ccefdcdf0b2/FlashEnglish/Sources/FlashEnglishApp.swift#L17"></span>Limit vertical whitespace to a single empty line; currently 6<br />
<code>vertical_whitespace</code> <code>FlashEnglishApp.swift:17</code></td>
    </tr>
  </tbody>
</table>